### DISCUSSION TITLE
How to disable the linking notification?

### DISCUSSION THREAD

**Discussion by Yanofly** [hass-emulated-hue/core#137] [Q&A]
I've already linked my emulated hub but every few minutes it shows up, how can I disable it without disabling the add-on?

**Comment by alexyao2015** [comment]
It only shows up when something is actively trying to connect to the hub.

**Reply by Yanofly** [reply]
My Yamaha AV Receiver is the source, is there anyway to block it?

I don't know why it is behaving like this:

`2021-03-15 12:28:24,377 DEBUG emulated_hue.discovery -- Serving SSDP discovery info to ('192.168.2.17', 46343)`

**Reply by alexyao2015** [reply]
No that's not it. You should see a log message saying something like the link button needs to be pressed. Directly before that log message should be the request that triggered it, along with the ip address.
